cmd/influxd/run/server.go

```diff
@@ -508,6 +513,41 @@ func (s *Server) reportServer() {
 	go client.Post("http://m.influxdb.com:8086/db/reporting/series?u=reporter&p=influxdb", "application/json", data)
 }
 
+// registerServer registers the server on start-up.
+func (s *Server) registerServer() error {
+	if s.enterpriseToken == "" {
+		return nil
+	}
+
+	clusterID, err := s.MetaStore.ClusterID()
+	if err != nil {
+		log.Printf("failed to retrieve cluster ID for registration: %s", err.Error())
+		return err
+	}
+
+	hostname, err := os.Hostname()
+	if err != nil {
+		return err
+	}
+
+	j := map[string]interface{}{
+		"cluster_id": clusterID,
+		"server_id":  s.MetaStore.NodeID(),
+		"host":       hostname,
+		"product":    "influxdb",
+		"version":    s.buildInfo.Version,
+	}
+	b, err := json.Marshal(j)
+	if err != nil {
+		return err
+	}
+
+	url := fmt.Sprintf("%s/?token=%s", s.enterpriseURL, s.enterpriseToken)
```

Comment:
Yeah, `go vet` caught that for me. It's fixed now. 

I'll fix up the API path now.